slice/IceLocatorDiscovery/IceLocatorDiscovery.ice

```diff
@@ -11,27 +11,29 @@
 
 #include "Ice/Locator.ice"
 
-/// IceLocatorDiscovery is an Ice plug-in that enables the discovery of IceGrid and custom locators via UDP multicast.
+/// IceLocatorDiscovery is an Ice plug-in that enables the discovery of Ice locators (such as IceGrid) via UDP
+/// multicast.
 module IceLocatorDiscovery
 {
-    /// The Ice lookup reply interface must be implemented by clients which are searching for Ice locators. Ice locator
-    /// implementations invoke on this interface to provide their locator proxy.
+    /// Represents a callback object implemented by IceLocatorDiscovery clients. It allows IceDiscoIceLocatorDiscovery
+    /// very clients to receive replies to Lookup requests.
```

Comment:
Seems you wanted to write something else `It allows IceDiscoIceLocatorDiscovery very clients`

Reply:
Fixed.